R/pkg/R/functions.R

```diff
@@ -67,6 +67,14 @@ createFunctions <- function() {
 
 createFunctions()
 
+#' @rdname functions
+#' @return Creates a Column class of literal value.
+#' @export
+lit <- function(x) {
```

Comment:
Can we define a S4 function which has argument signitures which are like `Any`? As far as I know, `setMethod` requires signatures of argument objects.
The argument of `lit` in Scala is `Any`. Otherwise, we should define multiple S4 function as follows.

```
setMethod("lit", signature("numeric"), .....)
setMethod("lit", signature("character"), .....)
...
```